Description: Fixes uses deprecated keyring.backend.$keyring
Author: Joe Gordon <joe.gordon0@gmail.com>
Bug-Debian: http://bugs.debian.org/728470
Origin: upstream, https://review.openstack.org/#/c/58364/
Last-Update: 2014-01-04

--- python-novaclient-2.15.0.orig/novaclient/shell.py
+++ python-novaclient-2.15.0/novaclient/shell.py
@@ -37,14 +37,6 @@ all_errors = ValueError
 try:
     import keyring
     HAS_KEYRING = True
-    try:
-        if isinstance(keyring.get_keyring(), keyring.backend.GnomeKeyring):
-            import gnomekeyring
-            all_errors = (ValueError,
-                          gnomekeyring.IOError,
-                          gnomekeyring.NoKeyringDaemonError)
-    except Exception:
-        pass
 except ImportError:
     pass
 
